File: python/Suma_Parzyste.py
#Program sumuje liczby parzyste w zakresie <0, 100>


def main(args):

    Początek = int(input("Podaj liczbę końcową: "))
    Końcówa = int(input("Podaj liczbę końcową: "))

    Essa = 0

    for Dab in range(Początek, Końcówa + 1):
        if Dab%2==0:
            Essa = Essa + Dab
    print(Essa)




    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/Suma_Parzyste.cpp

#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int Essa, Dab;
	
	int start, koniec;
	start = koniec = 0;
	
	
	cout << "Podaj liczbę początkową: ";
	cin >> start;
	
	cout << "Podaj liczbę końcową: ";
	cin >> koniec;
	
	Essa = 0;
	for (Dab = start; Dab < koniec+1; Dab++){
		if(Dab % 2 == 0)
			Essa = Essa + Dab;
	
	}
	cout << Essa;
	return 0;
}

